openassessment/templates/openassessmentblock/staff_area/oa_staff_grade_learners_assessment.html

```diff
@@ -2,21 +2,21 @@
 {% spaceless %}
 
     {% block body %}
-    <div class="ui-toggle-visibility__content">
+    <div class="staff__grade__form ui-toggle-visibility__content" data-submission-uuid="{{ submission.uuid }}">
```

Comment:
Github is showing this oddly. oa_staff_assessment.html actually moved to oa_staff_override_assessment.html (below). This is a new template for the "full grade" scenario.